templates/sag-cc-builder-repos/README.md

```diff
@@ -33,23 +35,16 @@ All supported Windows and UNIX platforms.
 
 ## Running as a standalone composite template
 
-To add product and fix repositories in Command Central:
+To add Software AG repositories to use with the Command Central Docker builder:
 
```

Comment:
There is a pre-requisite - the credentials for empower should have been created.
So we need either the command to apply sag-cc-creds template, or  a reference to its readme ( https://github.com/SoftwareAG/sagdevops-templates/tree/master/templates/sag-cc-creds.)